### ISSUE TITLE
Control of device (feature query/request)

### ISSUE THREAD

**Issue by wall0159** [atbrask/CMS50Dplus#3]
Is it possible to control the device via the serial interface? I would like to turn the device on and off, so that the laser is not running all the time (according to the manual, the device should not be worn for extended periods (more than 2 hours))

**Comment by atbrask** [comment]
No, I'm afraid not. To my knowledge there is no built-in way to turn the device on or off remotely from the PC. As far as I remember, there is no such feature in the bundled Windows software. Nothing is impossible, of course, but not without making hardware modifications to the device. I suppose you could add a small microcontroller that listens in on the serial line for custom turn on/off commands.
